source/Application.cpp

```diff
@@ -4,37 +4,38 @@
 ** Author Léo Lhuile
 **
 ** Started on  Thu Dec 28 21:54:14 2023 Léo Lhuile
-** Last update Thu May 30 08:59:00 2024 Léo Lhuile
+** Last update Sat Jul 26 21:26:53 2025 Léo Lhuile
 **
 ** Application.cpp : Implementation of Application class
 */
 
 #include "Application.hpp"
 #include "Debug.hpp"
+#include "ThemeManager.hpp"
 
 namespace FileShare::GUI {
     Application::Application()
-        : window(sf::VideoMode(800, 600), "FileShare")
-        , gui(window)
-    {
+        : window(sf::VideoMode(1200, 800), "FileShare")
+        , gui(window) {
     }
```

Comment:
I know we where talking about style earlier, and I said all `{` are on the same line, but kinda forgot about the constructors :sweat_smile: 

That's an exception : 
```suggestion
    Application::Application() : 
      window(sf::VideoMode(1200, 800), "FileShare"), 
      gui(window)
    {}
```
If there is nothing in the `{}` they go alone on their line. If there is something, then : 
```
    Application::Application() : 
      window(sf::VideoMode(1200, 800), "FileShare"), 
      gui(window)
    {
      something();
    }
```